examples/generate_QG_time_series.py

```diff
@@ -0,0 +1,84 @@
+"""Truth-twin and observation time series for QG (quasi-geostrophic) model."""
```

Comment:
I don't see the need for the "twin" wording in here (and elsewhere below). It's simply a forward simulation experiment. Also, even in DA settings, the wording used elsewhere in DAPPER is "synthetic truth" (with "twin" only sometimes put into parantheses or similar). Yes, it's my preference, but I stand by it, and as mentioned I even claim it's DAPPER "convention"

Reply:
Word, I'm removing this as it is indeed just a time series demonstration of the dynamical and observation models.